presto-main/src/main/java/com/facebook/presto/operator/OperatorStats.java

```diff
@@ -817,6 +817,11 @@ public OperatorStats add(Iterable<OperatorStats> operators)
                 joinProbeKeyCount);
     }
 
+    public static long convertDoubleToLong(double dataSize)
```

Comment:
I think we could remove this and just use a simple cast.  See https://docs.oracle.com/javase/specs/jls/se8/html/jls-5.html#jls-5.1.3